packages/terra-section-header/docs/README.md

```diff
@@ -0,0 +1,34 @@
+# Terra Section Header
+
+Section Header presentational component that provides an onClick callback and animateable status icon.
```

Comment:
This should be spelled 'animatable'.

Reply:
Good catch! 58a80795867b02ec7067de6d972b46cc2f78a633